src/llm.py

```diff
@@ -291,15 +292,15 @@ async def get_response_text(message: Message) -> Optional[str]:
         role = "assistant" if is_self else "user"
         data.append({"role": role, "content": msg})
 
-    response = await query_api(data)
-    if response:
-        # Remove "Busty: " prefix
-        prefix = f"{get_name(self_user)}: "
-        if response.startswith(prefix):
-            response = response[len(prefix) :]
-
-        return response
+    for x in range(config.LLM_RETRY):
```

Comment:
```suggestion
    for _ in range(config.LLM_RETRY):
```

If we're not using the variable, it's more common to use `_`.